README.md

```diff
@@ -23,25 +23,26 @@
 #### **JSON formatted example**
 ```
 {
-    {
-        "City": cityName,
-        "StartDate": startDate,
-        "EndDate": endDate, 
-        "AverageHighTemperature": averageHighTemperature,
-        "AverageLowTemperature": averageLowTemperature,
+    "response": {
+        "city": cityName,
+        "startDate": startDate(yyyy/mm/dd),
```

Comment:
Shouldn't we put it into the double quotes like ("yyyy/mm/dd")?

Reply:
Sure, we can do that

Reply:
@nuridak committed the new changes